vendors/espressif/boards/ports/ota_pal_for_aws/ota_pal.c

```diff
@@ -45,7 +45,7 @@
 #include "mbedtls/base64.h"
 #include "task.h"
 
-#define kOTA_HalfSecondDelay    pdMS_TO_TICKS( 500UL )
+#define OTA_HALF_SECOND_DELAY    pdMS_TO_TICKS( 500UL )
```

Comment:
From the description this is Renesas only PR but making some updates in Espressif port.

Reply:
Good catch, I reverted this change.